Makefile

```diff
@@ -18,10 +18,28 @@ all: iso
 debug ?= none
 net ?= none
 
+## Should we use the limine bootloader?
+use_limine ?= NO
+
 ## test for Windows Subsystem for Linux (Linux on Windows)
 IS_WSL = $(shell grep -is 'microsoft' /proc/version)
 
+## define $(bootloader)
+ifeq ($(use_limine),NO)
+	bootloader = grub
+
+	## Look for `grub-mkrescue` (Debian-like distros) or `grub2-mkrescue` (Fedora)
+	ifneq (,$(shell command -v $(GRUB_CROSS)grub-mkrescue))
+		GRUB_MKRESCUE = $(GRUB_CROSS)grub-mkrescue
+	else ifneq (,$(shell command -v $(GRUB_CROSS)grub2-mkrescue))
+		GRUB_MKRESCUE = $(GRUB_CROSS)grub2-mkrescue
+	else
+		GRUB_MKRESCUE = not-found
```

Comment:
I think we should restore the existing way this is done, since it simplifies the actual `grub` target.